src/main/java/org/folio/rest/impl/PatronBlockLimitsAPI.java

```diff
@@ -0,0 +1,118 @@
+    Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    PgUtil.get(PATRON_BLOCK_LIMITS, PatronBlockLimit.class,
+      org.folio.rest.jaxrs.model.PatronBlockLimits.class, query, offset, limit,
+      okapiHeaders, vertxContext, GetPatronBlockLimitsResponse.class, asyncResultHandler);
+  }
+
+  @Validate
+  @Override
+  public void postPatronBlockLimits(String lang, PatronBlockLimit entity,
+    Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    Errors errors = isEntityValid(entity);
+    if (errors != null) {
+      asyncResultHandler.handle(succeededFuture(PostPatronBlockLimitsResponse
+          .respond422WithApplicationJson(errors)));
+      return;
+    }
+
+    PgUtil.post(PATRON_BLOCK_LIMITS, entity, okapiHeaders, vertxContext,
+      PostPatronBlockLimitsResponse.class, asyncResultHandler);
+  }
+
+  @Validate
+  @Override
+  public void putPatronBlockLimitsByPatronBlockLimitId(String patronBlockLimitId,
+    String lang, PatronBlockLimit entity, Map<String, String> okapiHeaders,
+    Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+
+    Errors errors = isEntityValid(entity);
+    if (errors != null) {
+      asyncResultHandler.handle(succeededFuture(PutPatronBlockLimitsByPatronBlockLimitIdResponse
+          .respond422WithApplicationJson(errors)));
+      return;
+    }
+
+    PgUtil.put(PATRON_BLOCK_LIMITS, entity, patronBlockLimitId, okapiHeaders,
+      vertxContext, PutPatronBlockLimitsByPatronBlockLimitIdResponse.class, asyncResultHandler);
+  }
+
+  @Validate
+  @Override
+  public void getPatronBlockLimitsByPatronBlockLimitId(String patronBlockLimitId,
+    String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    PgUtil.getById(PATRON_BLOCK_LIMITS, PatronBlockLimit.class, patronBlockLimitId,
+      okapiHeaders, vertxContext, GetPatronBlockLimitsByPatronBlockLimitIdResponse.class,
+      asyncResultHandler);
+  }
+
+  @Validate
+  @Override
+  public void deletePatronBlockLimitsByPatronBlockLimitId(String patronBlockLimitId,
+    String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    PgUtil.deleteById(PATRON_BLOCK_LIMITS, patronBlockLimitId, okapiHeaders,
+      vertxContext, DeletePatronBlockLimitsByPatronBlockLimitIdResponse.class,
+      asyncResultHandler);
+  }
+
+  private Errors isEntityValid(PatronBlockLimit entity) {
+
+    return MAX_OUTSTANDING_FEEFINE_BALANCE_ID.equals(entity.getConditionId())
+      ? validateRangeForDoubleValueType(entity)
+      : validateRangeForIntegerValueType(entity);
+  }
+
+  private Errors validateRangeForDoubleValueType(PatronBlockLimit entity) {
+    Double limit = entity.getLimit();
+    if (limit > 0.1 && limit < 9999.99) {
```

Comment:
Description says:
> "Maximum outstanding fee/fine balance" may be blank or a currency value between 0.01 and 9999.99.

I sounds to me as `limit >= 0.1 && limit <= 9999.99`.

Reply:
thanks, updated